docs/sources/features/datasources/jaeger.md

```diff
@@ -44,4 +45,4 @@ Use the trace selector to pick particular trace from all traces logged in the ti
 
 ## Linking Trace ID from logs
 
-You can link to Jaeger trace from logs in Loki by configuring a derived field with internal link. See [Loki documentation]([Explore]({{< relref "./loki#derived-fields" >}})) for details.
+You can link to Jaeger trace from logs in Loki by configuring a derived field with internal link. See the [Derived fields]({{< relref "loki.md#derived-fields" >}}) section in the [Loki data source]({{< relref "loki.md" >}}) documentation for details.
```

Comment:
```suggestion
You can link to Jaeger trace from logs in Loki by configuring a derived field with internal link. See the [Derived fields]({{< relref "./loki.md#derived-fields" >}}) section in the [Loki data source]({{< relref "./loki.md" >}}) documentation for details.
```

Reply:
I think that there need to be a full path to the file that we want to link. In the current version, we were just missing `.md`.